### ISSUE TITLE
open options not implemented?

### ISSUE THREAD

**Issue by kitingChris** [matthiasg/node-acrobat#1]
```
var acrobat = require('acrobat');
acrobat.open('my.pdf', 
{
    page:12,
    pagemode:'none'
});
```

The pdf shows up but not on page 12 :(

**Comment by matthiasg** [comment]
Not sure I piped that through. Will look into that tomorrow ..

**Comment by matthiasg** [comment]
I hadnt finished that because this feature was implemented in another version of mine .. i will copy these features over tomorrow when i am in the office again

**Comment by kitingChris** [comment]
sounds promising :)

**Comment by matthiasg** [comment]
resolved in 0.1.0 14ffc8645038eee9cd3bab47d421ac65094ac899 also pushed to npm

running `node test.js` should open a preview first .. and then (once thats closed) the document with some options.

**Comment by kitingChris** [comment]
thanks
